sdk/iOS/src/MSSyncContextInternal.h

```diff
@@ -27,6 +28,8 @@
 
 -(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId completion:(MSSyncBlock)completion;
 
+-(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId pullSettings:(MSPullSettings *)pullSettings completion:(MSSyncBlock)completion;
```

Comment:
settings?  pullSettings seems a lil redundant.  Also, since this is an internal header, just change the existing sig.